### DISCUSSION TITLE
How to draw a shaded_dot which used in kmz

### DISCUSSION THREAD

**Discussion by thorly** [insarlab/MintPy#491] [General]
I modified the code of `save_kmz_timeseries.py` to fit formated txt file, and I used a non-shaded [circle](https://imgchr.com/i/r163wT) to make it, it looks like not very well. Mintpy's shaded_dot used in kmz is nice. Can you tell me how to draw this?

**Comment by yunjunz** [comment]
We use the native google earth icon directly http://kml4earth.appspot.com/icons.html. Your formatted text looks great!

**Reply by thorly** [reply]
Thanks
